packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue

```diff
@@ -347,6 +386,18 @@ export default {
       }, 1000);
       this.$emit("click:add-to-cart");
     },
+    handleSelectedColor(colorIndex) {
+      if (this.colors.length > 0) {
+        this.colors.map((color, i) => {
+          if (colorIndex === i) {
+            this.$emit("click:colors", color);
+          }
+        });
+      }
+    },
```

Comment:
I wonder, should the color picker be closed after choosing color on mobile? Maybe adding toggleColorPicker with isMobile condition here, could be good idea? Wdyt?